conda/recipe/meta.yaml

```diff
@@ -70,8 +70,8 @@ requirements:
     # for python 3.6 the latest protobuf conflicts with other dependencies
     # leave it as >= 3.18 and let conda just pick the latest that could be resolved
```

Comment:
The comment says to leave protobuf as >= 3.18 and you update it to 3.20. Do you want to update the comment or not update the version?

Reply:
Mhh. Maybe update the comment. We no longer test conda build for Python 3.6.

Reply:
I missed the comment, thanks for catching it. If it is not longer needed then I'll give the 3.20. a go.